paimon-common/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -194,6 +194,19 @@ public class CoreOptions implements Serializable {
                     .defaultValue(Duration.ofHours(1))
                     .withDescription("The maximum time of completed snapshots to retain.");
 
+    public static final ConfigOption<SnapshotExpireDeleteMode> SNAPSHOT_EXPIRE_MODE =
```

Comment:
Maybe we don't need this.

The concurrent deletion mode can be the default mode, which appears harmless.

Reply:
OK, I've removed this config option.